docs/articles/documentation/guides/continuous-integration/circleci.md

```diff
@@ -6,139 +6,224 @@ redirect_from:
+git checkout master
+git merge circleci-project-setup
 ```
 
-This command installs the `testcafe` and `testcafe-reporter-xunit` modules locally and adds them to `package.json`.
+## Step 4 - Add the `test` Script to package.json
+
+To specify how CircleCI should launch TestCafe, add the `test` script to the project's `package.json` file. Use the `testcafe` command in the script to run tests in Chrome in the headless mode. Add the `-r` flag to use the JUnit XML reporter.
 
 ```json
-{
-  "devDependencies": {
-      "testcafe": "*",
-      "testcafe-reporter-xunit": "*"
-  }
+"scripts": {
+  "test": "testcafe chrome:headless tests/**/* -r xunit:/tmp/test-results/testcafe/results.xml"
 }
 ```
 
-Provide a command to run tests in the `scripts` section.
+After you install the dependencies and specify the `test` script, `package.json` should look as follows:
 
 ```json
 {
   "scripts": {
-    "test": "testcafe chrome:headless tests/**/* -r xunit:/tmp/test-results/res.xml"
+    "test": "testcafe chrome:headless tests/**/* -r xunit:/tmp/test-results/testcafe/results.xml"
   },
   "devDependencies": {
-      "testcafe": "*",
-      "testcafe-reporter-xunit": "*"
+    "testcafe": "*",
+    "testcafe-reporter-xunit": "*"
   }
 }
 ```
 
-The `test` command runs tests from the `tests` directory in the headless Chrome. Reports are saved in the xUnit format at `/tmp/test-results/res.xml`.
+For more information on how to configure a test run with the `testcafe` command, see [Command Line Interface](../../reference/command-line-interface.md).
+
+## Step 5 - View Test Results
+
+Commit the changes to trigger the build. CircleCI Dashboard shows the results in the pipeline.
+
+Click the job name to see the details.
+
+![View the pipeline in the Dashboard](../../../images/circle-ci/view-pipeline.png)
+
+The `Steps` panel displays information about the completed job including the run duration, latest commit, and build log:
 
-Finally, push all changes to GitHub.
+![Job details in the Dashboard](../../../images/circle-ci/view-steps.png)
 
-## Step 3 - Running the Build
+The `Tests` panel displays test results. If errors occur, the report is shown.
 
-Use the **Start building** button to trigger a build.
+![View steps results in the Dashboard](../../../images/circle-ci/view-results.png)
 
-![Starting Build](../../../images/circle-ci/start-building.png)
+## Use Alternative Images
 
-## Step 4 - Viewing Test Results
+This guide uses the `cimg/node:lts-browsers`, an official CircleCI Docker image that comes with `lts` version of node and browser dependencies pre-installed. This image is in beta. If you run into problems with it, use one of the images listed below.
 
-Go to the **Builds** section and choose a build to view its results.
+### Use a TestCafe Image
 
-![Test Results](../../../images/circle-ci/fail-report.png)
+TestCafe is available as a preconfigured Docker image. The image has TestCafe, Chromium, and Firefox pre-installed.
```

Comment:
`TestCafe as an image`... That's not true I guess. "TestCafe is available as part of an image", "TestCafe is installed on an image"... But TestCafe is not an image itself.

Reply:
> TestCafe is available as a part of a preconfigured Docker image